database/seeds/ConferencesSeeder.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Carbon\Carbon;
+use Faker\Factory as Faker;
+use Illuminate\Database\Seeder;
+use Laracasts\TestDummy\Factory;
+
+class ConferencesSeeder extends Seeder
+{
+    public function run()
+    {
+        Conference::truncate();
+
+        $faker = Faker::create();
+        $user_ids = collect(User::lists('id'));
+        $conference_names = collect([
+            'MegaAwesomeCon',
+            'SuperPHP',
+            'ActiveRecordCon',
+            'ConCon',
+            'GoodJobFest',
+            'TightenFest',
+            'UltraMegaCon',
+            'ArbysCon',
+        ]);
+
+        foreach (range(1, 30) as $i) {
+            $starts_at = $faker->dateTimeBetween('now', '3 years');
```

Comment:
This could end up with weird overlaps between CFP and the conference. Do we care?